src/app.js

```diff
@@ -5,11 +5,96 @@
  */
 
 // override to native promise
- global.Promise = require('bluebird');
 
- const express = require('express');
- const http = require('http');
- const app = express();
+ global.Promise = require("bluebird");
 
- app.use('/', require('./views/index'));
- 
\ No newline at end of file
+const port = 3000;
+const favicon = require("serve-favicon");
+const express = require("express");
+const onFinished = require("on-finished");
+const moment = require("moment");
+const http = require("http");
+const path = require('path');
+const cluster = require("cluster");
+const Chalk = require('chalk').constructor;
+const os = require('os');
+const bodyParser = require("body-parser");
+const clk = new Chalk({ enabled: true });
+const numCPUs = os.cpus().length;
+
+const rstat = (s) => {
+    return s >= 500 ? clk.red.bold(s)
+        : s >= 400 ? clk.yellow.bold(s)
+            : s >= 300 ? clk.cyan.bold(s)
+                : s >= 200 ? clk.green.bold(s)
+                    : clk.gray.italic(s);
+}
+
+const log = (req, res, next) => {
+    const logRequest = () => {
+        const date = clk.cyan.bold(`[${moment().format('L')} @ ${moment().format('HH:MM:SS')}]`)
+        const ip = clk.blue.bold(`(${req.ip})`);
+        const status = rstat(res.statusCode || undefined);
+        const method = clk.bold(req.method);
+        const path = clk.magenta.bold(req.path);
+        console.log(date, ip, status, method, path);
+
+    }
+    onFinished(res, logRequest);
+    next();
+}
+
+if (cluster.isMaster) {
+    console.log(`Master ${process.pid} is running`);
+
+    for (let i = 0; i < numCPUs; i++) cluster.fork();
```

Comment:
this would probably slow down the system. As our IDE can go desktop mode or Cloud IDE mode, we need to specify when should we use this.